Main page

Anders-Bacon opened the pull request:
Currently i am working on the Carousel of the frontpage and making that work in a hard coded format. 
The same can be said for the category part of the page, the component has been made and can just be copied, but isnt really styled or anything yet

Commit:
Progress report 1 on Frontpage-gui

A lot of the HTML and CSS is in place, there is no functionality yet. There is established an connection to Font Awesome's library for icon usage.
There is currently lorum ipsum text in place of what should be there, just to keep tabs on how the html is placed.

Commit:
Merge branch 'main' into main-page-gui

AndersBaconThomsen committed:
Added navbar into our base layout

Added the navbar into our base layout file, so that it will appear on every page

AndersBaconThomsen committed:
Progress report 1 Navbar

Currently its a bootstrap navbar that have no functionality. there is also some custom css

Commit:
Progress report 2 - Navbar

The nav now looks nice and has no hover effects or wierd colors. The items itself are still not quite centered and there is no routing currently because i dont know if the specific pages it needs to route have been made

# Conflicts:
#	public/css/navbar.css
#	resources/views/components/navbar.blade.php

Commit:
Main page - 1

Beginning to form some part of the html and css

AndersBaconThomsen committed:
Navbar merge conflit

I dont know what happend but apparently it says a lot of things have been changed after a pull

Commit:
Merge branch 'main' into navbar

Commit:
Merge branch 'main' into main-page-gui

Commit:
HTML and CSS of front page

Currently there is only HTML and css and not that much content.

Commit:
Routing

The SMBD text and Home buttons now route to the home page

Commit:
Changes to HTML

i made the html look a little cleaner by removing tags that was now unnessecary and added a link to a javascript file

Commit:
Frontpage JS

The beginning of the frontpage javascript. It currently just alerts and logs if i have created a movie

AndersBaconThomsen committed:
Minor changes to frontpage

just added background color to differentiate better

AndersBaconThomsen committed:
Shorting text

Added a for loop that shortens down text if its bigger than 200 sign and replaces the rest with ...

Commit:
Frontpage 99% done

Frontpage now only needs the correct images and links for it to work. The individual extension is also done, but the images will be kept locally that they are not the correct ones.

AndersBaconThomsen committed:
Main page edits

now follows our component layout and classes for the carousel to make it more generalised, so that it can be called a library. There have also been chages to the JS file so that the refrences to the HTML ID's & classes match and again making the names more general.

AndersBaconThomsen committed:
Minor changes to navbar

NAvbar has a refrence to login and the route to login in web.php now has the name login

Anders-Bacon commented:
Currently the only thing the frontpage needs is a connection to the database so that the right content can be displayed

Commit:
Merge branch 'main' into main-page-gui

Commit:
Merge branch 'main' into navbar

Commit:
Preliminary images

These are just images to make sure my own individual extension is working

Commit:
Small edits

Just some small edits in order to fit database content properly in the view

Commit:
Individual Extension done

My individual extension is now done. i know have a controller that gathers the nesecary data from our databse. The JS file now fetches the JSON object and utelises a new route so that it knows where to fetch the data from.

Commit:
Main page categories

The main page now has some "categories" it has a for loop that just takes 5 movies and dispalys their img, but i cant refrence the trailer OR the category names because the databse is OVERLY COMPLICATED. I have added the css and html for it as well

Anders-Bacon commented:
Main page is done now, with both my indivdual extension and other things. Everything now has php variables and uses the data from the database

Commit:
Merge branch 'navbar' into main-page-gui

MidnightRocket reviewed (changes requested):
Separate individual from group work before merge can happen

Commit:
removal of individual

removal of individual for this branch just to add it to my completly own branch

Commit:
Removal of the JS to individual

removal of the individual js part, so that it can be put on its own branch

Commit:
deletion of images

Commit:
removal of individual css

Commit:
Individual is now completly seperated

MidnightRocket commented on app/Http/Controllers/categoryController.php:
Please rename to UpperCamel case

Anders-Bacon replied:
Done

Commit:
A miss of removal in the individual

needed to be removed

MidnightRocket commented on routes/web.php:
```diff
@@ -2,6 +2,8 @@
 use Illuminate\Support\Facades\Route;
 
+use App\Http\Controllers\CarouselController;
+use App\Http\Controllers\categoryController;
```
And rename here please

Anders-Bacon replied:
done

Commit:
Camelcase change

Commit:
Merge branch 'main' into main-page-gui

# Conflicts:
#	resources/views/components/layouts/base.blade.php
#	routes/web.php

MidnightRocket commented on resources/views/components/layouts/base.blade.php:
```diff
@@ -1,11 +1,15 @@
 <x-layouts.minimum {{ $attributes }}>
     <x-slot:head>
+        <link rel="stylesheet" href="{{asset('css/app.css')}}">
```
This line can be removed

MidnightRocket commented on app/Http/Controllers/categoryController.php:
Should be renamed to CamelCase

Anders-Bacon replied:
Done

Commit:
something idk

Commit:
duplicate

MidnightRocket committed:
Rename CategoryController

MidnightRocket committed:
Add signup button to navbar

MidnightRocket reviewed (approved):
Looks good now
